src/SprykerEco/Yves/Heidelpay/Theme/default/components/molecules/new-registration/new-registration.ts

```diff
@@ -1,28 +1,33 @@
 import Component from 'ShopUi/models/component';
 
 interface IFormElements extends HTMLFormElement{
```

Comment:
Please rename `IFormElements` into `IFormElement`. And add a space before `{`.

Reply:
fixed